package.json

```diff
@@ -322,6 +383,36 @@
                     "command": "azurerm-vscode-tools.sortVariables",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == variables",
                     "group": "arm-template"
```

Comment:
Rather than seeing "Insert Item..." and "Sort Template..." at every node of the tree view (that will get busy real quickly), Neil and I would like to see only the context menu items specific to that node on the node.  Then have a top-level command for sorting the entire template and a top-level Insert Item...  E.g. similar to this:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/6913354/80266130-394b5c80-864f-11ea-8a50-f49a87cdc82a.png">
See https://github.com/microsoft/vscode-azurestorage/blob/567a8c9ca43f91b1bad6f68133cc5da9866f56ea/package.json#L261 and https://github.com/microsoft/vscode-azurestorage/blob/567a8c9ca43f91b1bad6f68133cc5da9866f56ea/package.json#L333 for examples.  Feel free to copy any icons from that repo.  Not sure if codicons work here?  Let me know if you need help finding appropriate icons, or if you want help with implementing anything else.

Reply:
![image](https://user-images.githubusercontent.com/785949/80283476-18027300-8718-11ea-8573-594aaea2ada7.png)
![image](https://user-images.githubusercontent.com/785949/80283488-32d4e780-8718-11ea-88bc-ebfedf7daf66.png)
When I read the documentation I noticed that I could make the specific context menu items like (Sort Parameters, Insert Parameter...) inline. This makes the functionallity more discoverable in my opinion.
I downloaded the sort icon from https://www.svgrepo.com/svg/19599/sort-down because I could not find any similar icon in the repo you mentioned.